### DISCUSSION TITLE
Am I braindead ?

### DISCUSSION THREAD

**Discussion by R3xpook** [ScrapeGraphAI/Scrapegraph-ai#420] [Q&A]
I tried to use scrape graph on multiple newspaper sites using ollama and various model but the only thing that works is the ScriptCreatorGraph using deep seeker? Others LLM return me a list of image urls from the page , I'm also scraping Spanish pages, I don't know if the embedding model have influence on that but that still does not explain why deep seeker can generate a script for it

**Comment by itsperini** [comment]
Hey @R3xpook which LLMs have you tried? Do they always return the list of image urls? If you can give an example of websites I will try them ;)

**Reply by R3xpook** [reply]
I tried Gemma2 9b, Gemma7b,Llama3-8b ,phi3 mini and the 128k, deepseeker2 16b , I tried Spanish prompt I tried English prompts with Spanish one sometimes I get the title out the weird fact is that deepseeker2 actually generate a code to scrape the page so it technically means it works at least the code graph

**Comment by VinciGit00** [comment]
please update tot he new version
